daemon/start.go

```diff
@@ -158,6 +158,7 @@ func (daemon *Daemon) containerStart(container *container.Container, checkpoint
 
 	if resetRestartManager {
 		container.ResetRestartManager(true)
+		container.HasBeenManuallyStopped = false
```

Comment:
What's the reasoning behind changing this?

Reply:
we don't know which one of `kill` and 'libcontainerd.ExitEvent' comes first.

`containerStart()` in `exitEvent` should not overwrite `HasBeenManuallyStopped ` in such case